tests/diviner/test_diviner_model_export.py

```diff
@@ -321,9 +320,8 @@ def test_diviner_model_save_persists_requirements_in_mlflow_model_directory(
     mlflow.diviner.save_model(
         diviner_model=grouped_pmdarima, path=model_path, conda_env=diviner_custom_env
     )
-
-    saved_pip_req_path = os.path.join(model_path, "requirements.txt")
-    _compare_conda_env_requirements(diviner_custom_env, saved_pip_req_path)
+    saved_pip_req_path = pathlib.Path(model_path).joinpath("requirements.txt")
```

Comment:
```suggestion
    saved_pip_req_path = model_path.joinpath("requirements.txt")
```

`model_path` is already a `Path` object, right?